lib/commons/color/get-background-color.js

```diff
@@ -65,9 +77,48 @@ function calculateObscuringAlpha(elmIndex, elmStack) {
 }
 
 function elmOutsideParent(elm, bgElm, bgColor) {
```

Comment:
This function makes no sense to me either - why do we go through the effort of using elementsFromPoint if we are then going to ignore elements positioned outside their parent?